### PR TITLE
Removes Antag Tokens from the Xenoarch loot table

### PR DESCRIPTION
**Pull request by ghost**
[featureloss]

The end-game of a station department should not be to be an antagonist, no matter how rare it is.  I don't really think there is much more that needs to be said or reasoned on the matter.

This includes:
Ling vial
Stone mask
cult pamphlet 
syndicate beacon

:cl:
 * rscdel: Removed the antag tokens from xenoarcheology

### CODE REVIEW

File: code/modules/research/xenoarchaeology/artifact/artifact.dm [hunk 1]
```diff
@@ -15,15 +15,10 @@
 	artifact_id = generate_artifact_id()
 
 	artifact_find_type = pick(
-	5;/obj/machinery/syndicate_beacon,
```
Comment (jknpj) [inline code/modules/research/xenoarchaeology/artifact/artifact.dm:18]: This item existed since the times of olde and shall not be removed.

Comment (Jackreturnswithaveangenace) [reply code/modules/research/xenoarchaeology/artifact/artifact.dm:18]: the future is now old man

**Comment by DamianX** [comment]
What should the end-game of a station department be?

**Comment by ghost** [comment]
That is a rather open-ended question for a massive discussion that has nothing to do with this.  All I can say is that it shouldn't be antag tokens.

**Comment by tomacatx2** [comment]
> What should the end-game of a station department be?

level 50, max stats, all routes and CGs with the departmental waifus unlocked

**Comment by ghost** [comment]
For the sake of atomicity, I will move the warping claws to another PR

**Comment by SonixApache** [comment]
\>give syndies the ability to buy antag tokens
\>remove antag tokens from other jobs
what even is your endgame

**Comment by PrimeDSS13** [comment]
syndicate isn't a job, and it's when you already have an "antag token"
why should your job allow you to get free antag tokens

File: code/modules/research/xenoarchaeology/artifact/artifact.dm (hunk 1 above)
Comment (ghost) [reply code/modules/research/xenoarchaeology/artifact/artifact.dm:18]: old =/= good for the health of the game

I can comfortably say that the way xenoarcheology is perceived and how the job is treated as a way to just screw up the station and farm antag tokens is not a good thing for the server.

**Comment by ghost** [comment]
> 
> 
> >give syndies the ability to buy antag tokens
> >remove antag tokens from other jobs
> what even is your endgame

Taking these items out of the hands of a non-antag roundstart job  (a job that is perceived as an antag type of its own) and putting them into the hands of actual antags.

**Comment by iFlashYou** [comment]
Did a xenoarch dig up one of this four things and kill/antagonize you?

**Comment by ghost** [comment]
> 
> 
> Did a xenoarch dig up one of this four things and kill/antagonize you?

I was the Xenoarcheologist for the umteenth time.  I am willing to bet that I have the most finds for these items on the server.

**Comment by DrSnips** [comment]
> (a job that is perceived as an antag type of its own)

Citation needed.

**Comment by DrSnips** [comment]
I can support removing the warping claws but the antag things should just be reduced to the traitor beacon rather than all types.

**Comment by usnpeepoo** [comment]
remove xenoarch, otherwise nerf it to the ground. why? because science is for bitches. and bitches dont get shit.

**Comment by iFlashYou** [comment]
Server Poll?

**Comment by ghost** [comment]
I don't have a strong opinion on this since the person making the PR is also the only person I've seen obtain these items with any regularity even with the xenoarch buffs and I've never personally found one across several xenoarch rounds.

On the one hand, I don't know of it being a huge issue as compared to the low-key artifact grief (bringing ducks on station and conveniently leaving them where they can be activated, dragging floor/wall changing artifacts through the station, spamming time stops in medbay/on the shuttle, leaving dangerous/untested spawning pods in places) that has become endemic to the job now that people are able to dig up larges with great regularity. I have to wonder how many of the yes votes are riding the wave of xenoarch hate from all that grief considering they're still relatively rare finds.

On the other hand, dynamic gives us a ton of antags to begin with, so it's not like people desperately need additional free antag tokens, however rare they might be. It's also the only job that gets them (unless you count vault hunters/randos printing themselves digging tools as a job), which probably made a lot more sense when it required triangulation/powergaming/blind luck to find large digsites as opposed to getting ~10 chances a round. While neat, it's a strange incentive to have for doing your job, and let's be real, it exists so you can use those six roulette revolvers you dug up along the way even though you missed roundstart antag.

**Comment by anunknownentity03** [comment]
now this is grudge code if i ever seen it

**Comment by anunknownentity03** [comment]
actually, you know what I'll wait n see but I think it should be put to a poll

**Comment by gurfan** [comment]
I think the bigger issue is nonantag gunk artifact grief rather than digging up an antag token.

**Comment by PJB3005** [comment]
Idea: the xenoarch that digs the item up cannot use it but CAN give it to other people to use.

**Comment by Cieyl92** [comment]
> Idea: the xenoarch that digs the item up cannot use it but CAN give it to other people to use.

This is a fun idea. But it'd also just make xenoarcheologists even more hated than they already are. Maybe locking xenoarch traitor tokens behind the xenoarch already being a traitor might work?

**Comment by Dacendeth** [comment]
I think an alternative could be giving the items a chance to backfire, like when the syndie beacon used to explode most of the time when used the first time. For example the ling vial could turn you into a simple mob horrorform monster 50% of the time when you tried to use it.

**Comment by ShiftyRail** [comment]
@Dacendeth I think it would backfire by having xenoarch hand out "free" samples of their findings too often.

**Comment by xeroxemnas** [comment]
xenoarch is a pretty shit (and honestly dangerous) job since it's super easy to get killed on the roid by a vampire or something where no one will ever find you.

**Comment by PJB3005** [comment]
> xenoarch is a pretty shit (and honestly dangerous) job since it's super easy to get killed on the roid by a vampire or something where no one will ever find you.

In all my hours of playing xenoarch I don't think I've ever been backstabbed by an antag.

**Comment by yclatious** [comment]
Agreed with pjb. Killed like twice by miners, but beyond that I never ever got dunked on, I'd have been way more dunked on if I was on the station.

**Comment by Sooose** [comment]
I think I was killed by a vampire like once in all my time of playing xenoarchaeology, you're actually further removed from harm as the only thing that can kill you is goliaths if you don't know how to deal with them.

**Comment by Blithering** [comment]
I think so long as the other PR to make xenoarch slow the fuck down a bit passes, this isn't needed.

**Comment by kernikov** [comment]
Stop nerfing my gunk job.
Trying to clean sweep every small site I see has honed my mental math skills.

If you want to cut down on gunk, just nicely ask (read: force) Deity to code up something similar to threat value for a lot of the effects. Extended with nothing besides "feel good aura", black orbit with 3 warping claws and 2 stone masks.
The possibilities are endless.

**Comment by ghost** [comment]
image

Seems like a vote might be ideal.

File: code/modules/research/xenoarchaeology/artifact/artifact.dm (hunk 1 above)
Comment (jknpj) [reply code/modules/research/xenoarchaeology/artifact/artifact.dm:18]: By the love of carn don't make me leave this chair. It is the ONLY of them that isn't debug-tier gibantag and also gives the most common antag role. Not even worth the risk of using most of the time.

**Comment by DeityLink** [comment]
> The end-game of a station department should not be to be an antagonist

Antagonists are but a device to bring the round toward a conclusion. Xenoarchaeologists shouldn't play this role to get those tokens, but I don't think ways for players to acquire antag status mid-round are inherently bad and I don't think that rare xenoarch drops are a bad way to bring those about.

If they're that much of an issue currently I suggest you reduce their probability instead now that artifacts are more common.

**Comment by ghost** [comment]
> I don't think ways for players to acquire antag status mid-round are inherently bad and I don't think that rare xenoarch drops are a bad way to bring those about.

The job currently contributes more negatively than rounds than it does positively.  At best it is played as antag-lite, and at worse you are able to literally (in the case of the full crystal) able to end an entire round with a single click.  Why is this job so focused on screwing with the round AND being given free antag tokens?  The public opinion of xenoarcheologists is already absolutely awful because of the nature of the role and what it does to rounds.  Antag roles via the dynamic system to keep a round flow are not a bad thing.  Passively acquiring antag is not a bad thing, but granting the ability to actively farm for it, and making said farming the end-game of an entire department is a very bad thing.  Non-antag job roles should not have the ability to grind for a greater chance of being antagonist.

>Xenoarchaeologists shouldn't play this role to get those tokens

Except an environment has been made where the job does little more than screws up rounds and is additionally thrown free antag tokens.  Wishful thinking and passing things off as an administrative issue does nothing when the code is to the contrary.

**Comment by DeityLink** [comment]
I disagree with too many of your above statements to reply to them one by one but I'm kinda surprised how different your experience of xenoarchaology (both as a xenoarchaeologist or playing alongside one) is different from mine. The way I see it the problem comes from the player and not the job. Nice players will bring the best out of anything and bad players will bring the worst out of anything.

> but granting the ability to actively farm for it

I'll just single out this one statement and call it for the bullshit it is. The amount of large artifacts in a round is chosen randomly at roundstart and is static. It could be 5 or 20, but you cannot endlessly farm for them, it's be an even less reliable way to get antag than just getting selected by Dynamic.

**Comment by iFlashYou** [comment]
This shit was no problem for years and suddenly now it is?

**Comment by DeityLink** [comment]
> 
> 
> This shit was no problem for years and suddenly now it is?

Well because of me Large Artifacts ARE a lot more common (and antag tokens as well therefore) but I think we should just reduce their probability so they become as common as they were before if we really must change anything.

Also regardless of how easy it is to get large artifacts now, the job being now popular now also helps a lot.

**Comment by ghost** [comment]
> I'll just single out this one statement and call it for the bullshit it is. The amount of large artifacts in a round is chosen randomly at roundstart and is static. It could be 5 or 20, but you cannot endlessly farm for them, it's be an even less reliable way to get antag than just getting selected by Dynamic.

You absolutely can endlessly farm for them round-to-round with the intent of getting antag more than a normal player.  By completing the job you statistically have a higher chance of getting antag than someone who is in a different job slot.  These are things you can not dismiss.  Reliability isn't the issue.  The issue is that it shapes the job to be antag-lite/a path to being antag.  No other non-antag job does this and no non-antag job should be doing this.

image

This is not an uncommon sentiment.

**Comment by DeityLink** [comment]
For the record cult pamphlet specifically were a request and the PR got overwhelming support #27596

Once again I think the issue is more with the players than with the job. Also maybe with the fact that we have relatively little in terms of rules regarding self-antaging and the ways the game allows you to do it. As an admin I'd start adding notes to a player that regularly uses xenoarch to get antag and ruin the station. Finding one of those antag tokens doesn't mean you have to use them either, and that brings the question of antag tokens in Vaults and stuff.

**Comment by DeityLink** [comment]
So once again I suggest you halve their probability of spawning, and see if people are still bothered by it as much in a month.

**Comment by PJB3005** [comment]
To actually run the numbers on this, the actual weights make it so that every artifact has an approx ~1% chance (20/2020) of being an antag token. At least on box there appear to be an 10 artifacts per round on average, so that comes out to just a little below an, on average, **~10% chance of an antag token** *spawning* per round. You'll still have to actually dig them up though, let alone hope miners don't cuck you.

I can personally agree that number feels a little on the high side, so I'd be on board with reducing the drop chances perhaps.

**Comment by TGFH** [comment]
@Blithering 
> this isn't needed.

That's what I think about Xenoarch as a whole. 
Joking aside, I support the PR because the average Xenoarch is praying for antag so they get a chance to use all of the guns and larges they find every round. That shouldn't be encouraged with built in antag tokens regardless of their rarity.

**Comment by Cieyl92** [comment]
I think it'd be cool if xenoarch was more focused on getting artifacts that help out the station in some way.

**Comment by yclatious** [comment]
Cieyl, as a xenoarch player I fucking WISHED most artifacts could be useful and helpful. Do you know how much I'd love a healing/battery charger/touch based artifact that I could even hand out on portable battery emitters? Those are beyond rare because first of all, the large pool is bloated with other shit that isn't just randomized larges and is barely useful at all.

Then we have the gunk of even testing the large artifacts with randomized triggers, some of them obnoxious as shit like not staring at them, the room being on fire, oxygen, them needing to be splashed with thermite, so on. Some triggers being easier to get is great and all, but god knows that most of these are a pain in the ass to activate to begin with. What they do is even worse 99% of the time because it ranges from mostly useless to actual on touch Ei Nath that spawns monsters. God, I wished there were more positive ones, but the average thing I have is either clockworkify/cultist (which is grief as fuck), gas spawners which never ever end well, useless fluff, are too obnoxious to trigger like requiring coins or straight up just dissapointing things like shifting some of the floor tiles around them.

So many larges do barely anything useful at all, and then you not only have to extract them off the roid (which soose wants to gut reliably by making everyone do fucking triangulation instead of having an Alden Sasparova, which will probably pass) , pray to the heavens it isn't another corgi pod/suicide by driving into gibtonite large drill/shitty space pod/whatever, but then you also have to sacrifice your infant to have the large piece of junk not be utterly terrible, a pain in the ass to activate and for the crew to not imediately try and lynch your ass whenever you bring it aboard. 

The chances are fucking rigged against us, and mind you, I am in *favour* of removing antag tickets. We as xenoarchs can't do much good _because we barely fucking have anything good at all._ It's like if virology existed only to make the biggest poz load neo dark death and had maybe tricord spawn once every 5 minutes after turning you into a tarjaran.

**Comment by Jackreturnswithaveangenace** [comment]
I think Yclat's post has some really important points. Take away some of the antag stuff (after all, Xenoarch can still get interesting guns and roulette revolvers), make it more likely that artifacts have positive effects or at least that there are more positive effects to roll.

**Comment by DeityLink** [comment]
@Cieyl92
 
> I think it'd be cool if xenoarch was more focused on getting artifacts that help out the station in some way.

That's literally how I play Xenoarch 🤷 

Rare seeds for xenobio and the bar, exotic gas artifacts for atmos, SMES charging artifacts for Engineering and silicons, Healing artifacts for medbay, alien antibodies and beneficial symptoms for Virology, infinite meat for the kitchen from cryopods, etc

Unlike shitter xenoarch players if I find a fucking Floor Tile Changing artifact, or Clockwerk artifact, I won't bring them to the station.

**Comment by Blithering** [comment]
A floor tile artifact can help seal a breach in a pinch, if you get the create effect instead of the change one. Key is to use one of the batteries instead of dragging the artifact around.

**Comment by DeityLink** [comment]
And once again the deciding factor is the skill and intent of the player.

**Comment by Kurfursten** [comment]
Giving this 4 more days to try to get a clear majority, doesn't look good

**Comment by Rubylips** [comment]
They're extremely rare and totally RNG based. They're fine how they are. The only qualm I have with them is that xenoarch is really the only source of them. That does give xenoarch a "trying to be antag" flavor.

**Comment by Eneocho** [comment]
Instead of removing them or nerfing the spawn rate further, make it send a warning à la cult overspreading, so that using the antag token is still possible, but the xenoarch has to take into consideration all the station will know of their newly acquired antag status.
>We have detected an abnormal energy signature associated with cultist activity on [SPECIFIC AREA OF USE] (cult)


>The NMV Icarus intercepted a scrambled transmission on [SPECIFIC AREA OF USE]. Encryption patterns match with Syndicate communications. (syndie beacon)


>A piercing screech of a thousand souls goes through you. You feel it come from [GENERAL AREA OF USE], god help us all. (ling)


>A wave of unholy power coming from [GENERAL AREA OF USE] makes you feel your blood pulse. Your neck feels oddly naked.

**Comment by ghost** [comment]
The server has spoken, it is fine as is.

**Pull request closed without merging**